lib/foreman_remote_execution/engine.rb

```diff
@@ -53,6 +53,7 @@ class Engine < ::Rails::Engine
 
     initializer 'foreman_remote_execution.register_plugin', before: :finisher_hook do |_app|
       Foreman::Plugin.register :foreman_remote_execution do
+        register_global_js_file 'routes'
```

Comment:
can we rename the file to `global` in case we will want to put more things that are not routes in there?

Reply:
Done!